sdk/redis/Azure.ResourceManager.Redis/src/Generated/RedisLinkedServerWithPropertyData.cs

```diff
@@ -27,12 +27,16 @@ public RedisLinkedServerWithPropertyData()
         /// <param name="linkedRedisCacheId"> Fully qualified resourceId of the linked redis cache. </param>
         /// <param name="linkedRedisCacheLocation"> Location of the linked redis cache. </param>
         /// <param name="serverRole"> Role of the linked server. </param>
+        /// <param name="geoReplicatedPrimaryHostName"> The unchanging DNS name which will always point to current geo-primary cache among the linked redis caches for seamless Geo Failover experience. </param>
+        /// <param name="primaryHostName"> The changing DNS name that resolves to the current geo-primary cache among the linked redis caches before or after the Geo Failover. </param>
         /// <param name="provisioningState"> Terminal state of the link between primary and secondary redis cache. </param>
-        internal RedisLinkedServerWithPropertyData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, ResourceIdentifier linkedRedisCacheId, AzureLocation? linkedRedisCacheLocation, RedisLinkedServerRole? serverRole, string provisioningState) : base(id, name, resourceType, systemData)
+        internal RedisLinkedServerWithPropertyData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, ResourceIdentifier linkedRedisCacheId, AzureLocation? linkedRedisCacheLocation, RedisLinkedServerRole? serverRole, string geoReplicatedPrimaryHostName, string primaryHostName, string provisioningState) : base(id, name, resourceType, systemData)
```

Comment:
There are breaking changes in the interfaces (e.g. parameters are inserted before existing parameters). Here are 2 options:
  - Add manual codes to provide backward compatibility, see compute for reference: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations
- Or you need to bump the major version